piezo_web_app/setup.py

```diff
@@ -0,0 +1,35 @@
+from PiezoWebApp import version
+from setuptools import setup, find_packages
+
+setup(
```

Comment:
This is probably unnecessary as we are always going to be deploying the app in a pod and thus won't need to package it for standalone deployment

Reply:
Fair point! I'm tempted to say that we should keep it in: it's little effort to maintain and keeps our options open. We should mention this in the documentation, though.

Reply:
Agreed, it shouldn't need much maintaining but we should definitely add it to the documentation